cfr.json

```diff
@@ -1559,5 +1559,167 @@
     "template": "update_action",
     "targeting": "firefoxVersion >= 69 && localeLanguageCode in [\"en\",\"fr\",\"de\"] && ((currentDate|date - profileAgeCreated) / 86400000) >= 4 && ((currentDate|date - profileAgeCreated) / 86400000) < 15",
     "id": "WNP_MOMENTS_1"
+  },
+  {
+    "id": "SOCIAL_TRACKING_PROTECTION",
+    "template": "cfr_doorhanger",
+    "priority": 1,
+    "content": {
+      "layout": "icon_and_message",
+      "category": "cfrFeatures",
+      "anchor_id": "tracking-protection-icon-box",
+      "skip_address_bar_notifier": true,
+      "bucket_id": "CFR_SOCIAL_TRACKING_PROTECTION",
```

Comment:
You might want to make `bucket_id` more specific here, since we won't report the `id` in release.

For example, change it to `TRACK_PROTECTION_SOCIAL_TRACKING`.

Reply:
Same to other messages in this PR.